### ISSUE TITLE
53450813 - Bayonetta

### ISSUE THREAD

**Issue by voltagex** [xenia-project/game-compatibility#96]
benvanik/xenia@fcdd15d1
http://marketplace.xbox.com/en-AU/Product/BAYONETTA/66acd000-77fe-1000-9115-d80253450813

Doesn't load after 30+ seconds, screen stays green.

Logs at https://github.com/voltagex/xenia-logs/blob/master/53450813.txt

**Comment by Razzile** [comment]
Possible duplicate of https://github.com/xenia-project/game-compatibility/issues/43 ?

**Comment by voltagex** [comment]
One's a demo, this isn't.

**Comment by voltagex** [comment]
Still state-nothing as of benvanik/xenia@720f8b0

![image](https://cloud.githubusercontent.com/assets/159567/14954607/1ae04240-10b7-11e6-9225-79c563d07d3b.png)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/245041/xenia.zip)


Can I remove the original log? I'd like to pull down the repo soon.

**Comment by uselesscunt** [comment]
Bayonetta plays opening video and displays main menu. Eternal loading screen

https://github.com/benvanik/xenia/commit/4aaca8cbf3288e5334cfafccea9cbfe0d5875e7f
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/328973/xenia.zip)
![bayo1](https://cloud.githubusercontent.com/assets/17132199/16287990/6abe9fa2-38b5-11e6-8667-6703f08dd997.png)
![bayo2](https://cloud.githubusercontent.com/assets/17132199/16287995/6d5dff00-38b5-11e6-8c4e-fc3c17b7620a.png)

**Comment by voltagex** [comment]
state-nothing with benvanik/xenia@bc0ddbb05a. Crashes before starting to render graphics. Occasionally you get half a second of audio

```
>	xenia.exe!vkCmdDraw(VkCommandBuffer_T * commandBuffer, unsigned int vertexCount, unsigned int instanceCount, unsigned int firstVertex, unsigned int firstInstance) Line 1894	C
 	xenia.exe!xe::gpu::vulkan::VulkanCommandProcessor::IssueDraw(xe::gpu::PrimitiveType primitive_type, unsigned int index_count, xe::gpu::CommandProcessor::IndexBufferInfo * index_buffer_info) Line 581	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacketType3_DRAW_INDX_2(xe::RingBuffer * reader, unsigned int packet, unsigned int count) Line 1103	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacketType3(xe::RingBuffer * reader, unsigned int packet) Line 585	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacket(xe::RingBuffer * reader) Line 451	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecuteIndirectBuffer(unsigned int ptr, unsigned int count) Line 410	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacketType3_INDIRECT_BUFFER(xe::RingBuffer * reader, unsigned int packet, unsigned int count) Line 746	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacketType3(xe::RingBuffer * reader, unsigned int packet) Line 552	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacket(xe::RingBuffer * reader) Line 451	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecuteIndirectBuffer(unsigned int ptr, unsigned int count) Line 410	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacketType3_INDIRECT_BUFFER(xe::RingBuffer * reader, unsigned int packet, unsigned int count) Line 746	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacketType3(xe::RingBuffer * reader, unsigned int packet) Line 552	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePacket(xe::RingBuffer * reader) Line 451	C++
 	xenia.exe!xe::gpu::CommandProcessor::ExecutePrimaryBuffer(unsigned int read_index, unsigned int write_index) Line 388	C++
 	xenia.exe!xe::gpu::CommandProcessor::WorkerThreadMain() Line 156	C++
 	xenia.exe!xe::gpu::CommandProcessor::Initialize::__l2::<lambda>() Line 53	C++
```

[53450813.log.gz](https://github.com/xenia-project/game-compatibility/files/478149/53450813.log.gz)

**Comment by druidamortis** [comment]
I've got the same problem with https://github.com/benvanik/xenia/commit/a3c2b8b3f5b12762214556b44abc0c152cafc27b 
I managed to get an error message with visual studio: Unhandled exception at 0x000000000109E1B4 (nvoglv64.dll) in xenia.exe: 0xC0000005: Access violation reading location 0x000000000000019C.
log: [xenia.zip](https://github.com/xenia-project/game-compatibility/files/494845/xenia.zip)

**Comment by voltagex** [comment]
Can you check what your driver version is?

**Comment by Parovozik** [comment]
OpenGL (Boot OK) http://imagehost.spark-media.ru/i4/585C0E0B-45D9-92FA-7BF3-4E891391731A.jpg
Vulkan (Boot failed. Crash) http://imagehost.spark-media.ru/i4/585C0E0B-45D9-92FA-7BF3-4E891391731A.jpg
LOG http://slidex.narod.ru/xenia_Bayonetta_VK_log.log

**Comment by voltagex** [comment]
@Parovozik you can attach your log by dragging it to the issue comment.
[xenia_Bayonetta_VK_log.zip](https://github.com/xenia-project/game-compatibility/files/495605/xenia_Bayonetta_VK_log.zip)

**Comment by druidamortis** [comment]
@voltagex 372.90 (latest version)

**Comment by Rhodan81** [comment]
Now goes ingame with heavy visual glitches. Tested with https://github.com/benvanik/xenia/commit/4866a4a9663822b532b8833da12eca1a54d35c3c.

**Comment by Parovozik** [comment]
Tested on (https://github.com/benvanik/xenia/commit/16cff62d36201651bfc998a623516f6cda721fa0)

# Issues:
Vulkan: Partially broken graphics and after some time render go in grey screen but game still works (sound, sub-text). Some objects invisible. Not works transparency for some effects and textures
OGL: Picture much better but very slow performance

# LOGs:
[Bayonetta_VK.txt](https://github.com/xenia-project/game-compatibility/files/999528/Bayonetta_VK.txt)
[Bayonetta_OGL.txt](https://github.com/xenia-project/game-compatibility/files/999530/Bayonetta_OGL.txt)

# Video:
[YouTube_Bayonetta_Vulkan](https://www.youtube.com/watch?v=SRSYTXK5bLg)

# Screenshots:
Vulkan:
![desktop 05 14 2017 - 18 29 58 03 mp4_snapshot_01 10_ 2017 05 14_19 10 18](https://cloud.githubusercontent.com/assets/19775644/26035700/29f8025e-38d9-11e7-9e0c-3f072b788e8f.jpg)
Vulkan:
![desktop 05 14 2017 - 18 29 58 03 mp4_snapshot_01 30_ 2017 05 14_19 11 01](https://cloud.githubusercontent.com/assets/19775644/26035701/2fee38d6-38d9-11e7-81e2-126f35648e8d.jpg)
OpenGL:
![desktop 05 14 2017 - 18 31 52 04 mp4_snapshot_05 27_ 2017 05 14_19 11 14](https://cloud.githubusercontent.com/assets/19775644/26035704/364d5158-38d9-11e7-808e-87d34b6ff338.jpg)
OpenGL:
![desktop 05 14 2017 - 18 31 52 04 mp4_snapshot_07 50_ 2017 05 14_19 11 24](https://cloud.githubusercontent.com/assets/19775644/26035706/3a8ad86c-38d9-11e7-8578-f752b9cc58ed.jpg)

# Labels:
gpu-corrupt-drawing
gpu-missing-drawing
state-gameplay

**Comment by Parovozik** [comment]
Used last master (https://github.com/benvanik/xenia/commit/2a52156edf4ecc02d9083c31d495e0a927d26438) + (https://github.com/benvanik/xenia/issues/779) + (https://github.com/benvanik/xenia/issues/767)
Game works much better now, but still partially broken graphics and requires CPU fix (Opcode_Splat as disable). Prologue works is enough good, Part 1 - playable but more graphics glitches, Part 2 - half playable, some place as white screen.
Performance - perfect
Sound - OK
Save-States - OK
![xenia bayonetta mp4_snapshot_02 25_ 2018 03 01_00 30 30](https://user-images.githubusercontent.com/19775644/36815462-cdcfa8c2-1ceb-11e8-99ce-4d396259f975.jpg)
# Video:
[YouTube_Bayonetta_Vulkan](https://www.youtube.com/watch?v=uFHHIuWE6P4)

**Comment by zemzema** [comment]
Tested on (https://github.com/benvanik/xenia/commit/6b8a34c9ba7e4886d7c42de1658682acfffd0a11)

With AMD graphic card it's look like:

![screenshot 48](https://user-images.githubusercontent.com/33085810/36989740-e2295d9e-20a2-11e8-941c-4c24d8c0883a.png)

**Comment by zemzema** [comment]
Fixed with version:
Tested on (https://github.com/benvanik/xenia/commit/170ab850491713720e72247aa6895a7c2725562e)

# Log:
[bayonetta.zip](https://github.com/xenia-project/game-compatibility/files/2001787/bayonetta.zip)

**Comment by FitterSpace** [comment]
### Tested on https://github.com/xenia-project/xenia/commit/f748e5af49062fd78d098acdaf045183f29faf36

I don't have much to add to zemzema's post, but I figured I'd test this game on the current master and provide a video and some screenshots to go with it.

### Video: 
https://youtu.be/5Ob8NWXA72c

### Screenshots:
![](https://i.gyazo.com/d47cbc50f89e57c64519593eff7c1f39.png)
![](https://i.gyazo.com/729d1e8c4e70961c74e3d2487bb0fa0f.jpg)
![](https://i.gyazo.com/2ad8afe8a7b8dc6277360d54563b7213.jpg)

The lighting in the first part of the prologue (shown in the first screenshot) is totally screwed up, but it gets better after that. A lot of the textures look pretty funky, but I haven't seen many graphical issues that would affect gameplay once you get past the intro. The performance is also really good. During the prologue I managed a consistent 60 frames per second, with rare drops in the framerate if there were explosions on screen. The framerate was a bit lower in chapter 1, probably because the level design is more complex and there is more stuff happening during that part of the game. Note that my performance is better than the video shows because recording the screen slows down my computer a bit. The sound and music are also really clear.

### Log File (53.7 MB):
https://www.dropbox.com/s/zke2cm31kavxmyi/Bayonetta.zip?dl=0

**Comment by Umit007** [comment]
The brightness increases in this part of the scene. I don't know if it's because of the game itself or the emulator. 
XENIA VERSION (FEB 17 2022) Direct3D 12
![Adsız](https://user-images.githubusercontent.com/101103924/157717318-7a6efc49-e1d3-41bc-aaee-e4377296741f.jpg)

**Comment by Umit007** [comment]
Chapter XIV for to some have problem.

Stage Starting before
![Adsız](https://user-images.githubusercontent.com/101103924/157752773-2af87d85-6253-4d62-9bfb-51dca661bed0.jpg)

Pass some timing after
![Adsız](https://user-images.githubusercontent.com/101103924/157753496-f2d38103-15a7-49e0-8d46-47e71e50d607.jpg)

**Comment by insaneninja117** [comment]
Tested on master@6b45cf844 (Feb 17 2022) build from a blank save upto Chapter 14 where the same issue as described above appears. It is possible to button mash your way to the end of that segment but then the game will be stuck in an infinite loading screen. Overall the game up until that point is pretty playable and generally gameplay suffers from few graphical glitches, but cutscenes are more problematic.

PC Specs:
Windows 7 SP1 Professional
Intel Core i3-6100
AMD Radeon Pro WX3100
8GB DDR3

![Bayonetta 1](https://user-images.githubusercontent.com/38947445/160296948-e58826cc-808f-42d8-a273-dd3fd9e46eeb.jpg)
![Bayonetta 2](https://user-images.githubusercontent.com/38947445/160296968-8638fa33-aa65-4788-a654-90397f09de44.jpg)
![Bayonetta 3](https://user-images.githubusercontent.com/38947445/160297012-046c012d-7df7-48ce-8938-bb36dab313cf.jpg)
![Bayonetta 4](https://user-images.githubusercontent.com/38947445/160297020-1810c3ee-f2db-40ed-981a-d816ad3719c2.jpg)
![Bayonetta 5](https://user-images.githubusercontent.com/38947445/160297027-aab11e1d-1103-45d5-87a9-70e32bc2ef04.jpg)
![Bayonetta 6](https://user-images.githubusercontent.com/38947445/160297038-700e7bb2-3cd7-4d2e-b2f0-c720725401ea.jpg)
![Bayonetta 7](https://user-images.githubusercontent.com/38947445/160297053-613e6aaf-2666-4ca8-a5d2-ba38021dcbd4.jpg)
![Bayonetta 8](https://user-images.githubusercontent.com/38947445/160297062-40ed355b-d0a4-4897-b540-e54c02deb9f3.jpg)
![Bayonetta 9](https://user-images.githubusercontent.com/38947445/160297170-2d430064-be8b-44a5-b4a6-ca7aa91c1cde.jpg)
![Bayonetta 10](https://user-images.githubusercontent.com/38947445/160297183-bb278fec-a9a0-4e2e-b9be-973bda99d772.jpg)
![Bayonetta 11](https://user-images.githubusercontent.com/38947445/160297189-db743040-fab0-4698-86cf-645b601048c6.jpg)
[Xenia Log Bayonetta Chapters 7-9.zip](https://github.com/xenia-project/game-compatibility/files/8358310/Xenia.Log.Bayonetta.Chapters.7-9.zip)
[Xenia Log Bayonetta Chapter 14.zip](https://github.com/xenia-project/game-compatibility/files/8358328/Xenia.Log.Bayonetta.Chapter.14.zip)

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/540
